docs/fsharp/language-reference/value-options.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Value Options (F#)
+description: Learn about the F# Value Option type, which is a struct version of the Option type.
+ms.date: 06/16/2018
+---
+
+# Value Options
+
+The Value Option type in F# is generally used when the following two circumstances hold:
+
+1. A scenario is appropriate for an [F# Option](options.md).
+2. Using a struct provides a performance benefit in your scenario.
+
+It's important to note that not all performance-sensitive scenarios are "solved" by using structs, and you must consider the additional cost of copying when using them instead of reference types. However, in large F# programs it is common to have many Optional types instantiated and flowed through hot paths, where structs can sometimes yield better overall performance over the lifetime of a program.
+
+## Definition
+
+Value Option is defined as a [struct discriminated union](discriminated-unions.md#struct-discriminated-union) that is nearly identical to reference Option:
+
+```fsharp
+[<StructuralEquality; StructuralComparison>]
+[<CompiledName("FSharpValueOption`1")>]
+[<Struct>]
+type ValueOption<'T> =
+    | ValueNone: 'T voption
+    | ValueSome: 'T -> 'T voption
+
+    member Value : 'T
+
+and 'T voption = ValueOption<'T>
+```
+
+Value Option conforms to structural equality and comparison. The main difference is that the compiled name, type name, and case names all indicate that it is a value type.
```

Comment:
names --> name (assuming that there's only one case name)

Reply:
There are two case names